src/components/calculatorStyle.scss

```diff
@@ -0,0 +1,62 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.main-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 0;
+  row-gap: 0;
+  margin: auto;
+  margin-top: 100px;
+  padding: 0;
+  width: 300px;
+  background-color: white;
+  grid-template-areas:
```

Comment:
- [ ] OPTIONAL: I believe there is still something to do with your UI design compared to the given design. You can make changes when you have spare time. :dancers: 
![image](https://user-images.githubusercontent.com/64894927/128974458-f21e5197-8b2e-43a7-ac34-1c34a48e5e44.png)         
![image](https://user-images.githubusercontent.com/64894927/128974487-27f53518-d684-4603-90c2-0587a19ba09c.png)